sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Services/WebPubSubService.cs

```diff
@@ -21,16 +18,19 @@ internal class WebPubSubService : IWebPubSubService
         public WebPubSubService(string connectionString, string hubName)
         {
             _serviceConfig = new ServiceConfigParser(connectionString);
-            var url = !string.IsNullOrEmpty(_serviceConfig.Port) ? $"{_serviceConfig.Endpoint}:{_serviceConfig.Port}" : _serviceConfig.Endpoint;
-            _client = new WebPubSubServiceClient(new Uri(url), hubName, new AzureKeyCredential(_serviceConfig.AccessKey));
+            UriBuilder endpoint = _serviceConfig.Port == 80 ?
+                new UriBuilder(_serviceConfig.Endpoint) :
+                new UriBuilder(_serviceConfig.Endpoint.Scheme, _serviceConfig.Endpoint.Host, _serviceConfig.Port);
```

Comment:
Is it intended that we changed the check from port != null to port != 80?

Reply:
Not needed.